apps/tailwind-components/components/form/Field.vue

```diff
@@ -35,30 +35,54 @@ function validate(value: columnValue) {
 
   formFieldInput.value.validate(value);
 }
+
+const refData = ref();
```

Comment:
I am on the fence if we want to do this here. How will you implement search?
In my opinion it is best to organize the loading (lazily) per input so we can differentiate behaviors.
Which will be different between select, checkbox, and ontology.

Reply:
I thought you said to remove refData here?

Reply:
and that we would only focus on the listbox in isolation?